rmgpy/solver/liquid.pyx

```diff
@@ -173,11 +174,17 @@ cdef class LiquidReactor(ReactionSystem):
         modelSettings is not used here, but is needed so that the method
         matches the one in simpleReactor.
         """
-        # Set the maximum unimolecular rate to be kB*T/h
-        unimolecular_threshold_rate_constant = 2.08366122e10 * self.T.value_si
+        families = getDB('kinetics').families.keys()
+
         # Set the maximum bi/trimolecular rates based on the Smoluchowski and Stokes-Einstein equations
-        bimolecular_threshold_rate_constant = 22.2 * self.T.value_si / self.viscosity
-        trimolecular_threshold_rate_constant = 0.11 * self.T.value_si / self.viscosity
+        unimolecular_threshold_rate_constant = []
+        bimolecular_threshold_rate_constant = []
+        trimolecular_threshold_rate_constant = []
+        for k in xrange(len(families)):
+            unimolecular_threshold_rate_constant.append(2.08366122e10 * self.T.value_si)
```

Comment:
Couple more suggestions here:
- Keep the comment for unimolecular threshold from before: `# Set the maximum unimolecular rate to be kB*T/h`
- Import constants and use constants.kb/constants.h for the unimolecular threshold
- Make sure to convert these to numpy arrays like in the method simple.pyx, which is now required by the type declaration in base.pyx
- I still don't like importing from the database module here. I can think of two alternatives: 1) since the simulate method now has num_families, you can pass it to this method, 2) if the arrhenius fits are also loaded for liquid simulations, you can check the length of the dictionary from model settings. If you choose 1, make sure to update the function signature for simple.pyx as well.

Reply:
- Keeping the comment for unimolecular threshold from before: `# Set the unimolecular rate to kB*T/h`.
- Using now constants.kB and constants.h for unimolecular threshold.
- Using numpy operations to assign values.
- Chose alternative 2) to avoid database module loading.
- To be consistent and potentially faster I also use numpy operations in simple.pyx `get_threshold_rate_constants`